pages/docs/tutorials/streetlights.md

```diff
@@ -88,7 +90,7 @@ Now lets move all the way to the `channels` section. This section is used to des
       operationId: onLightMeasured`}
 </CodeBlock>
 
-In this example, `light/measured` is the channel name your API will `publish` to. The `operationId` property, describes what will be the name of function or method that takes care of this functionality in the generated code. The `payload` property is used to understand how the event should look like when publishing to that channel:
+In this example, `light/measured` is the channel name your API will `subscribe` to (i.e, to interact with the API you `publish` to the broker). The `operationId` property, describes what will be the name of function or method that takes care of this functionality in the generated code. The `payload` property is used to understand how the event should look like when publishing to that channel:
```

Comment:
```suggestion
In this example, `light/measured` is the channel name the Streetlight API will `subscribe` to (i.e, to interact with the Streetlight API you `publish` to the broker). The `operationId` property, describes what is the name of function or method that takes care of this functionality in the generated code. The `payload` property is used to understand how the event should look like when publishing to that channel:
```

I suggest this change because initially `your API` meant not Streetlight, but an app that interacts with it - so it was correct, but I understand why it is confusing. Thus suggest to not write `your API` at all. Thoughts?